udata/static/less/forms.less

```diff
@@ -210,6 +210,19 @@ form {
     }
 }
 
+.form-upload-fields {
```

Comment:
I think it's already globally defined.
See ``udata/static/js/form/common.js`` and http://getbootstrap.com/css/#forms-control-validation

We should keep it uniform by reusing the same classes and addind the background feedback globally.